Storage/SQL/SQLiteHistory.swift

```diff
@@ -124,12 +124,16 @@ private let topSitesQuery = "SELECT * FROM \(TableCachedTopSites) ORDER BY frece
 extension SQLiteHistory: BrowserHistory {
     public func removeSiteFromTopSites(site: Site) -> Success {
         if let host = site.url.asURL?.normalizedHost() {
-            return db.run([("UPDATE \(TableDomains) set showOnTopSites = 0 WHERE domain = ?", [host])])
-                >>> { return self.refreshTopSitesCache() }
+            return self.removeHostFromTopSites(host)
         }
         return deferMaybe(DatabaseError(description: "Invalid url for site \(site.url)"))
     }
 
+    public func removeHostFromTopSites(host: String) -> Success {
```

Comment:
I'm working on adding the default SuggestedSites now and instead of having to handle two different models all the way down to the view layer like the old topsites I can convert both suggestedSites and Sites into TopSiteItems and compare them that way. I'll have to make very minimal changes to the view layer and it won't need to know anything about suggestedSites. 

sidenote: I was really hoping to just take the asset name of a SuggestedSite icon and set it to the faviconURL and have SDWebImage load it. I'd have to make 0 changes that way. SDWebImage does support loading files locally. But sadly not from xcassets 😢